docs/source/AdministratorGuide/DIRACSites/ComputingElements/index.rst

```diff
@@ -121,9 +94,13 @@ of the *ComputingElement* is located in the inside the corresponding site sectio
   }
 
 
-This is the general structure in which specific CE descriptions are inserted. The CE configuration
-is part of the general DIRAC configuration. It can be placed in the general Configuration Service
-or in the local configuration of the DIRAC installation. Some CE parameters are confidential, e.g.
+This is the general structure in which specific CE descriptions are inserted.
+The CE configuration is part of the general DIRAC configuration
+It can be placed in the general Configuration Service or in the local configuration of the DIRAC installation.
+
+Additional info can be found :ref:`here <resourcesComputing>`.
```

Comment:
I think this gives a warning, becayse "here" is used in different places as the target name
cf. https://github.com/DIRACGrid/DIRAC/commit/f4c364ede4e079416fadbcefe4a1d50acbdba3e8#diff-769d9b4761c42439b1d1c6574dcf3d8dR669

```suggestion
Additional info can be found :ref:`here <resourcesComputing>`__.
```